src/components/LeftSidebar/ConversationsList/Conversation.vue

```diff
@@ -372,6 +373,9 @@ export default {
 
 		// forward click event
 		onClick(event) {
+			EventBus.$emit('conversationClicked', {
```

Comment:
seems this isn't used anywhere, maybe it was part of a previous attempt.
remove ?